packages/unzip.rb

```diff
@@ -3,25 +3,18 @@
 class Unzip < Package
   description 'UnZip is an extraction utility for archives compressed in .zip format (also called \'zipfiles\').'
   homepage 'http://www.info-zip.org/UnZip.html'
-  version '1.6_1-1'
+  version '1.6_1-2'
   compatibility 'all'
   source_url 'https://downloads.sourceforge.net/project/infozip/UnZip%206.x%20%28latest%29/UnZip%206.0/unzip60.tar.gz'
   source_sha256 '036d96991646d0449ed0aa952e4fbe21b476ce994abc276e49d30e686708bd37'
 
   binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/unzip-1.6_1-1-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/unzip-1.6_1-1-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/unzip-1.6_1-1-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/unzip-1.6_1-1-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    aarch64: 'e72843178559c924341a969fe9560b56e00b4dd539085e853f0de9602240970a',
-     armv7l: 'e72843178559c924341a969fe9560b56e00b4dd539085e853f0de9602240970a',
-       i686: 'c166df14f26735426dbbbd9bfddcfbf1940057d4b3ef2713adc75f4225312af1',
-     x86_64: 'b3ef4ace8bc32020363d00183eaf19ec5f8358180292216558927f937da1daea',
   })
 
   depends_on 'compressdoc' => :build
+  depends_on 'libiconv'
```

Comment:
This is already a core dependency.  Here is my take.  There is a file conflict between glibc and libiconv.  If you install libiconv after glibc, then the libiconv version is used.  The problem is on a fresh install, libiconv is installed first.  I think what we need to do is remove the conflicting library file from glibc.

Reply:
100%. I suggest making the iconv.h in libiconv primary and have the glibc remove iconv.h during the installation stage.

Reply:
I was backwards here. glibc in linux INCLUDES the iconv functions, so linux distributions use the iconv.h from glibc. So we need to REMOVE the iconv.h from libiconv. 

Here's my suggestion to fix this issue without requiring the adding of libiconv dependencies, and yes this will require a package rebuild to remove the offending ```iconv.h```:
```
require 'package'

class Libiconv < Package
  description 'GNU charset conversion library for libc which does not implement it.'
  homepage 'https://www.gnu.org/software/libiconv/'
  version '1.16-3'
  compatibility 'all'
  source_url 'https://ftpmirror.gnu.org/libiconv/libiconv-1.16.tar.gz'
  source_sha256 'e6a1b1b589654277ee790cce3734f07876ac4ccfaecbee8afa0b649cf529cc04'


  def self.build
    system "./configure #{CREW_OPTIONS} --enable-static --enable-relocatable --enable-extra-encodings"
    system 'make'
  end

  def self.install
    system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
    # Remove iconv.h which conflicts with same file from glibc
    FileUtils.rm "#{CREW_DEST_PREFIX}/include/iconv.h"
  end
end
```

Reply:
(Of course, install that, and then reinstall glibc, and then compiles will stop complaining about libiconv.)